MLproject

```diff
@@ -0,0 +1,10 @@
+name: yiqing-test
+
+docker_env:
+  image: grointelligence/worker
```

Comment:
For someone to pull this docker image, they would need to log into DockerHub with Gro credentials.

Reply:
Let me add a comment here to make that clear, so we know this is a hard constraint.